src/Computaria/Tardis/Factory.php

```diff
@@ -26,7 +26,7 @@ public static function grow(
 
     private static function createProxyFactoryIfNecessary(ProxyManager\AbstractBaseFactory $proxyFactory=null)
     {
-        if (false === is_null($proxyFactory)) {
+        if (!is_null($proxyFactory)) {
```

Comment:
yes! but do u think that type of condition can be impacted on performance or something else? i did not understand what u mean